find_green_build

```diff
@@ -100,6 +100,9 @@ FLAGS_verbose=1
 MYLOG=/tmp/$PROG.log
 common::logfileinit $MYLOG 10
 
+# Set default API/GIT roots
+gitlib::set_api_roots $PROJECT_REPO
```

Comment:
why is this script getting this arg but not the others?

Reply:
Oversight. Removed.

This whole change was added as a convenience so I didn't have to type --project_repo=istio/$repo with each invocation of relnotes.  The info is available when running it, so take advantage of it.  It may also be useful as other tools are added and want the added convenience.